packages/@webex/plugin-meetings/test/unit/spec/meeting/muteState.js

```diff
@@ -509,3 +509,221 @@ describe('plugin-meetings', () => {
+        assert.notCalled(muteLocalTrackSpy);
+        assert.notCalled(setServerMutedSpy);
+        assert.calledOnceWithExactly(applyClientStateToServerSpy, meeting);
+        assert.isFalse(muteState.state.client.localMute);
+      });
+
+      it('#muteLocalTrack', async () => {
+        setup(mediaType, true);
+        setupSpies(muteState, mediaType);
+
+        muteState.init(meeting);
+
+        const serverMutedSpyCall1 = setServerMutedSpy.getCall(0);
+        const serverMutedSpyCall2 = setServerMutedSpy.getCall(1);
+
+        assert.calledOnceWithExactly(applyUnmuteAllowedToTrackSpy, meeting);
+        assert.calledOnceWithExactly(muteLocalTrackSpy, meeting, true, 'remotelyMuted');
+        assert.equal(muteState.ignoreMuteStateChange, false);
+        assert.equal(serverMutedSpyCall1.calledWithExactly(true, 'remotelyMuted'), true);
+        assert.equal(serverMutedSpyCall2.calledWithExactly(true, 'remotelyMuted'), true);
+        assert.calledOnceWithExactly(applyClientStateToServerSpy, meeting);
+        assert.isFalse(muteState.state.client.localMute);
+      });
+
+      it('#muteLocalTrack tracks undefined', async () => {
+        setup(mediaType, true, false, false);
+        setupSpies(muteState, mediaType);  
+        muteState.init(meeting);
+        assert.calledOnceWithExactly(applyUnmuteAllowedToTrackSpy, meeting);
+        assert.calledOnceWithExactly(muteLocalTrackSpy, meeting, true, 'remotelyMuted');
+        assert.equal(muteState.ignoreMuteStateChange, false);
+      });
+
+      describe('#handleLocalTrackMuteStateChange', () => {
+
+        afterEach(() => {
+          sinon.restore();
+        });
+
+        it('checks when ignoreMuteStateChange is true', () => {
+          setup(mediaType);
+          muteState.ignoreMuteStateChange= true;
+          muteState.state.client.localMute = false;
+
+          const spy = sinon.spy(muteState, 'applyClientStateToServer');
+          muteState.handleLocalTrackMuteStateChange(meeting, true);
+          assert.notCalled(spy);
+          assert.isFalse(muteState.state.client.localMute);
+        });
+
+        it('tests localMute - true to false', () => {
+          setup(mediaType);
+          muteState.state.client.localMute = true;
+
+          const spy = sinon.spy(muteState, 'applyClientStateToServer');
+          muteState.handleLocalTrackMuteStateChange(meeting, false);
+          assert.equal(muteState.state.client.localMute, false);
+          assert.calledOnceWithExactly(spy, meeting)
+        });
+
+        it('tests localMute - false to true', () => {
+          muteState.state.client.localMute = false;
+
+          const spy = sinon.spy(muteState, 'applyClientStateToServer');
+          muteState.handleLocalTrackMuteStateChange(meeting, true);
+          assert.equal(muteState.state.client.localMute, true);
+          assert.calledOnceWithExactly(spy, meeting)
+        });
+      });
+
+      describe('#applyClientStateLocally', () => {
```

Comment:
it's not clear to me what is the purpose of the tests inside this describe from the their titles. Are you sure you meant "ignoreMuteStateChange" in the titles and not sdkOwnsLocalTrack? Can you make them more descriptive?